django/chat/templates/chat/components/chat_message.html

```diff
@@ -1,8 +1,9 @@
 {% load static %}
 {% load i18n %}
 
-<div class="message-outer {% if message.is_bot %}bot{% elif not file_upload and not message.num_files %}user{% endif %}{% if message.awaiting_response %} awaiting-response{% endif %}"
-     id="message_{{ message.id }}">
+<div class="message-outer {% if message.is_bot %}bot{% elif not file_upload and not message.num_files %}user{% endif %} {% if message.awaiting_response %}awaiting-response{% endif %} {% if swap_oob %}show-all{% endif %}"
+     id="message_{{ message.id }}"
+     {% if swap_oob %}hx-swap-oob="true"{% endif %}>
```

Comment:
As I mentioned earlier, this template is now rendered at the end of htmx_stream, which ends the streaming response (since the outer message, which previously contained the SSE element, has been replaced with one that does not have a SSE element).